train_gpt2.c

```diff
@@ -1054,8 +1075,7 @@ int main() {
 
     // some memory for generating samples from the model
     unsigned long long rng_state = 1337;
-    const int gen_max_length = 64;
-    int gen_tokens[gen_max_length];
+    int gen_tokens[GEN_MAX_LENGTH];
```

Comment:
Windows C11 impl wasn't happy with this. Not sure what C standard you're using, but typically GCC/Clang are more strict so this was surprising

Reply:
If I remember correctly, VLA's are a C99 feature that Microsoft does not support with their compiler. Be great if they did but they don't. I would recommend adding -Werror=vla to the GCC compiler options to avoid this in the future.